docs/platforms/react-native/performance/instrumentation/time-to-display.mdx

```diff
@@ -0,0 +1,87 @@
+
+Time to Display consists of two parts, Time To Initial Diplay (TTID) and Time To Full Display (TTFD). TTID measures the first time screen was displayed, while TTFD measures the time it takes for a requested content to apear.
+
+## Automatic Time To Initial Display for React Navigation
+
+To measure Time to Initial Display of React Navigation Screens, you need `Sentry.ReactNavigationInstrumentation` instance with `enableTimeToInitialDisplay` option enabled.
+
+<Include name="react-native-react-navigation-setup.mdx" />
+
+### Notes
+
+- To learn more about Sentry's React Navigation integration, see the [Automatic Instrumentation page](/platforms/react-native/performance/instrumentation/automatic-instrumentation).
+
+- The measured value of Time to Initial Display starts when React Navigation dispatches the navigation event and ends when the next frame after the native Screen component initialization is rendered.
+
+- If the screen is animated, the measured value of Time to Initial Display will include the time it takes for the animation to complete. Except for JavaScript driven animations on iOS.
+
+- Time to Initial Display is only available for React Navigation version 5 and above.
+
+## Time To Full Display
+
+To measure the time when a screen is fully displayed, you can use `Sentry.TimeToFullDisplay` component.
+
+```jsx {tabTitle: <TimeToFullDisplay />}
+import * as Sentry from "@sentry/react-native";
+import { useState, useEffect } from "react";
+import { View, Text } from "react-native";
+
+function MyComponent() {
+  const [data, setData] = useState(null);
+
+  useEffect(() => {
+    fetch('https://example.com/data')
+      .then(response => response.json())
+      .then(data => setData(data));
+  }, []);
+
+  const shouldRecordFullDisplay = data !== null;
+  return (
+    <View>
+      <Sentry.TimeToFullDisplay record={shouldRecordFullDisplay} />
+      {data ? <Text>{data}</Text> : <Text>Loading...</Text>}
+    </View>
+  );
+}
+```
+
+## Time To Initial Display Overwrite
+
+When the automatically captured Time to Initial Display doesn't meet your requirements, you can overwrite it by using `Sentry.TimeToInitialDisplay` component.
+
+```jsx {tabTitle: <TimeToInitialDisplay />}
+import * as Sentry from "@sentry/react-native";
+import * as React from "react";
+import { View } from "react-native";
+
+function MyComponent() {
+  return (
+    <View>
+      <Sentry.TimeToInitialDisplay record={true} />
+    </View>
+  );
+}
+```
+
+## Notes
+
+- The `Sentry.TimeTo*` components behave the same as `<></>` when wrapped around other components.
```

Comment:
```suggestion
- The `Sentry.TimeToInitialDisplay` and `Sentry.TimeToFullDispaly` components behave the same as `<></>` when wrapped around other components.
```
Since it's only two components, let's just spell it out